dockerfiles/hpcc/templates/_helpers.tpl

```diff
@@ -99,7 +71,8 @@ data:
 {{- /* Add standard volume mounts for a component */ -}}
 {{- define "hpcc.addVolumeMounts" -}}
 volumeMounts:
```

Comment:
volumeMounts: heading should move into components, and be changed to addConfigMapVolume ?

Reply:
Yes, once we agree the direction is where we want to go, there's more to be done...